modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
 
-[NOTE]
---
-Starting from Mule versions 4.2.0 and 3.9.3, you can configure the Mule runtime engine to forward events to both a third-party system, like ELK and Splunk, and to Anypoint Platform by adding an additional property to the `wrapper.conf` file:
+To configure multiple consumers, add the following line to the `wrapper.conf` file:
 
 [source,java,linenums]
 ----
 wrapper.java.additional.<n>=-Danypoint.platform.analytics_multiple_consumers_enabled=true
 ----
---
-
-== Integrating API Analytics with Splunk and ELK
-
-Once you've configured your API gateway, you can now return to Runtime Manager and see that your servers have some additional options in their menu.
-
-image::sending-data-from-arm-to-external-monitoring-software-api-analytics.png[]
-
-You can now set up the sending of API analytics to both Splunk and ELK, you configure exactly in the same way as you do when sending business events to them. See <<Integrating with an ELK Stack, Integrating with an ELK Stack>> and <<Integrating with Splunk, Integrating with Splunk>>.
 
-== Payload Formats
 
-If you intend to export your payload's contents to be logged, keep in mind that not all formats can be exported. See xref:about-logging-of-payload-formats.adoc[About Logging of Payload Formats] for a full reference.
+=== Configure API Analytics to Connect with Splunk or ELK
 
-== Encrypting Passwords
 
-Assign a master password to your Mule instance or your API gateway standalone instance when launching them. If you don't, when you set up your credentials for external applications using the Runtime Manager UI, the credentials are stored as plain text in the `conf/mule-agent.yml` file. This is not recommended for security reasons. Instead, launch Mule or API gateway runtime with an extra argument that is then used to encrypt these passwords when storing them in the YAML file.
+After you've configured your API gateway, configure Splunk or ELK in Runtime Manager:
 
-=== Mule
-
-----
-{MULE_HOME}/bin/mule -M-Dmule.agent.configuration.password=myMasterPassword
-----
-
-or on Windows:
-
-----
-{MULE_HOME}\bin\mule.bat -M-Dmule.agent.configuration.password=myMasterPassword
-----
-
-
-=== API Gateway - Deprecated
+. From Anypoint Platform, select *Runtime Manager*.
+. Click the *Servers* tab.
+. Click the *Type* column to display the details pane for the server to configure.
+. Click *Manage Server*.
+. Click the *Plugins* tab:
++
+.The arrow shows the *API Analytics* options on the server *Plugins* tab.
+image::hybrid-export-api-analytics.png[API Analytics options on the server Plugins tab.]
 
-----
-{MULE_HOME}/bin/gateway -M-Dmule.agent.configuration.password=myMasterPassword
-----
 
-or on Windows:
+. Under *API Analytics*, enable *ELK* or *Splunk*.
 
-----
-{MULE_HOME}\bin\gateway.bat -M-Dmule.agent.configuration.password=myMasterPassword
-----
+. Click the gear icon and continue with step 8 in the following sections to complete the configuration:
```

Comment:
```suggestion
. Click the gear icon and continue with Step 8 in the following sections to complete the configuration:
```
Do you suppose that there could come a time when the step numbers will change and the text be out of sync?